src/components/color_picker/color_picker_entry.ts

```diff
@@ -0,0 +1,71 @@
+import { Component } from "@odoo/owl";
+import { SpreadsheetChildEnv } from "../../types";
+import { css } from "../helpers";
+import { _lt } from "./../../translation";
+import { ColorPicker } from "./color_picker";
+
+interface Props {
+  currentColor: string | undefined;
+  toggleColorPicker: () => void;
+  showColorPicker: boolean;
+  onColorPicked: (color: string) => void;
+  icon: string;
+  dropdownDirection?: "left" | "right" | "center";
+  tooltipText?: string;
+  disabled?: boolean;
+}
+
+css/* scss */ `
+  .o-color-picker-entry {
+    display: inline-block;
+    position: relative;
+    .o-tool {
+      display: flex;
+      align-items: center;
+      margin: 2px;
+      padding: 0 3px;
+      border-radius: 2px;
+    }
+    .o-tool.active,
+    .o-tool:not(.o-disabled):hover {
+      background-color: rgba(0, 0, 0, 0.08);
+    }
+    .o-with-color > span {
+      border-bottom: 4px solid;
+      height: 16px;
+      margin-top: 2px;
+      cursor: pointer;
+    }
+  }
+
+  .o-cf-cell-is-rule .o-color-picker-entry span {
+    cursor: default;
+  }
+`;
+
```

Comment:
there is one css rule that seems to be used for every color picker but that is not here: 
```css
      .o-with-color {
        .o-line-item:hover {
          outline: 1px solid gray;
        }
      }
```

Reply:
@rrahir On master branch I only found class `o-line-item` is used to set borders in top bar, not with color picker. Did I miss something?